src/doc/rustdoc/src/what-is-rustdoc.md

```diff
@@ -34,6 +34,9 @@ the main page is located in `doc/lib/index.html`. If you open that up in
 a web browser, you will see a page with a search bar, and "Crate lib" at the
 top, with no contents.
 
+You can also use `cargo doc` to generate documentation for the whole project.
+See [Using rustdoc with Cargo](#rustdoc-cargo).
```

Comment:
The correct anchor is `#using-rustdoc-with-cargo`.

Reply:
I have applied your suggestions, but the autocheck failed for some reason.

Reply:
No you didn't.

Reply:
Sorry for the confusion. Please see edited commit.
@rustbot ready